### DISCUSSION TITLE
Pull Request Approval can't be done at font size I need

### DISCUSSION THREAD

**Discussion by rlove** [community/community#129227] [Accessibility]
### Select Topic Area

Product Feedback

### GitHub Feature Area

Pull Requests

### Problem
I am unable to see the approve options on a code review in a pull request as the dialog won't scroll.

### Personal ADA abilities
I have low vision one eye and complete loss of vision in the other eye.  I don't use JAWS or a screen reader (at this point), I just typically need large fonts. 

I have a monitor that has 3440x1400 resolution with Scale in the Windows OS set to 200%.    
The Zoom on my browser is set at 150%  

### Screen shot examples
Screenshot of images 
<img width="1720" alt="Unable to See Approval full dialog to see." src="https://github.com/community/community/assets/1970827/06974018-2ce0-4e1f-8035-cf5b83128331">

I always have to change zoom temporarily to see full dialog as when I use "F11" to view full screen, it will still not show the full content, and can only see the radio options on the screen and not the post button.

<img width="1720" alt="Version with F11 pressed" src="https://github.com/community/community/assets/1970827/5fbaa0f4-d83a-40b6-bcc7-663c7d3dc3d4">

#### Side Note   
I am thankful that GitHub website has been so easy to work with given my personal ADA concerns. This is the only thing I have to adjust on the site each time to do my work.

**Comment by cehfisher** [comment]
Hello @rlove - thank you for providing this feedback! We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Comment by cehfisher** [comment]
Hello @rlove - there is a new [commit details experience](https://github.blog/changelog/2025-03-12-refreshed-commit-details-page-now-generally-available) that should help resolve the font resizing issue you reported. If it does help, we can use the same approach on other pages. Please test it out and let us know what you think!
